pages/studentInstanceQuestionHomework/studentInstanceQuestionHomework.ejs

```diff
@@ -51,6 +51,10 @@
                   <td><%- include('../partials/questionValue', {value: instance_question.current_value});  %></td>
```

Comment:
The enclosing `<table>` should be `<table class="table table-condensed two-column-description-no-header">`, like this: https://github.com/PrairieLearn/PrairieLearn/blob/master/pages/studentInstanceQuestionExam/studentInstanceQuestionExam.ejs#L60